vmdb/app/models/mixins/process_tasks_mixin.rb

```diff
@@ -0,0 +1,123 @@
+module ProcessTasksMixin
+  extend ActiveSupport::Concern
+
+  module ClassMethods
+    # Processes tasks received from the UI and queues them
+    def process_tasks(options)
+      raise "No ids given to process_tasks" if options[:ids].blank?
+      if options[:task] == "refresh_ems" && respond_to?("refresh_ems")
+        refresh_ems(options[:ids])
+        msg = "'#{options[:task]}' initiated for "\
```

Comment:
Oh I see, yeah, in this case you do need it, but you can also just put it on one line (I personally dislike the line continuator in all cases).